vendor/sunnyvalley/+POST_INSTALL.post

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+if [ -f /usr/local/etc/pkg/repos/SunnyValley.conf.temp ]; then
```

Comment:
I'm not very enthusiastic about the approach here tied to post-install point.

Reply:
When i start health check process, got mismatch warning about /usr/local/etc/pkg/repos/SunnyValley.conf.

Reply:
losing or truncating SunnyValley.conf for any reason will not allow for recovery. I know the same is true for the current package but at least we have this exact hint that the file is an/the issue. So you are missing a trigger to rewrite/regenerate to which you say you can run from your code but it further elevates the problem at hand in that the code is not robust enough :)

Reply:
i noticed this., if when os-sunnyvalley remove , Sunnyvalley.conf  will not be deleted. :(

Reply:
If we could define a variable similar to the ABI variable for our NODEUUID and use this variable within the repository, our problem would be solved.

Reply:
I can bring in the sample/shadow keyword approach from core. that would fix removal of the .conf file at least.

Publishing an environment is relatively problematic to work reliably. That's the main reason we went with an embedded approach for the repo file (like you are doing) but also had to build some padding in opnsense-update to cope with that situation. I'm not sure what the best way forward is but I'm happy to help this get shipped in a robust way.